### ISSUE TITLE
management interface netmask is not correctly populated (ceos)

### ISSUE THREAD

**Issue by sulrich** [srl-labs/containerlab#396]
when using ceos-lab images the netmask for the management interface is being
set to `/0`.  this obviously causes some problems.  when poking around in the
code it looks like the `/{{.MgmtIPv4PrefixLength}}` element in the template is
not getting populated from whatever is gleaned from the container inspection. 

unfortunately, i don't have the time to dig into this further, but here's some
potentially useful info.  note, this issue applies to all of the containers
which startup.  i've included the relevant snippets from a sample
configuration for brevity.  the output is consistent across the balance of the
container. 

version info 
```shell
version: 0.13.0
 commit: fa230f6
   date: 2021-04-13T17:37:22Z
 source: https://github.com/srl-labs/containerlab
```

no modifications have been made to the stock template included in `0.13.0`

relevant node configuration from the topo file (sample)

```yaml
    ep2:
      kind: ceos
      image: ceos:latest
      mgmt_ipv4: 172.20.20.22
```

relevant config - immediately after startup
```
ep2#sh run int ma0
interface Management0
   ip address 172.20.20.22/0
```

relevant containerlab inspection
```text
+----+---------------+--------------+-------------+------+-------+---------+-----------------+--------------+
| #  |     Name      | Container ID |    Image    | Kind | Group |  State  |  IPv4 Address   | IPv6 Address |
+----+---------------+--------------+-------------+------+-------+---------+-----------------+--------------+
... snipped ... 
|  7 | clab-g-rr-ep2 | 58671d05b459 | ceos:latest | ceos |       | running | 172.20.20.22/24 | NA           |
... snipped ... 
```

docker container inspection (network settings)

```json
         "NetworkSettings": {
            "Bridge": "",
            "SandboxID": "f2add9e6f7146a8dbfb337540d331049d2c31485f6796670eb29083e3f4ac292",
            "HairpinMode": false,
            "LinkLocalIPv6Address": "",
            "LinkLocalIPv6PrefixLen": 0,
            "Ports": {},
            "SandboxKey": "/var/run/docker/netns/f2add9e6f714",
            "SecondaryIPAddresses": null,
            "SecondaryIPv6Addresses": null,
            "EndpointID": "",
            "Gateway": "",
            "GlobalIPv6Address": "",
            "GlobalIPv6PrefixLen": 0,
            "IPAddress": "",
            "IPPrefixLen": 0,
            "IPv6Gateway": "",
            "MacAddress": "",
            "Networks": {
                "clab": {
                    "IPAMConfig": {
                        "IPv4Address": "172.20.20.22"
                    },
                    "Links": null,
                    "Aliases": [
                        "58671d05b459"
                    ],
                    "NetworkID": "689fe1214a9adec850525daddf151156982f29f7e384a3c77d3260fac1b85c54",
                    "EndpointID": "77352514bed5f972a4cd208ff0a9d1b3de3ae7847d690f7d1b869e5af79d074b",
                    "Gateway": "172.20.20.1",
                    "IPAddress": "172.20.20.22",
                    "IPPrefixLen": 24,                << netmask available
                    "IPv6Gateway": "",
                    "GlobalIPv6Address": "",
                    "GlobalIPv6PrefixLen": 0,
                    "MacAddress": "00:1c:73:de:b0:e9",
                    "DriverOpts": null
                }
            }
        }
    }
```

**Comment by hellt** [comment]
thanks @sulrich 
that is indeed a bug that will be addressed in #397 
if you want, you can checkout this branch and test the fix.

**Comment by sulrich** [comment]
that works. thanks!
